.ci/matrix_job_release.yaml

```diff
@@ -20,14 +20,20 @@ steps:
   - name: Build Plugin
     agentSelector: "{nodeLabel: 'SWX-CI-DOCKER'}"
     run: |
+      if [ -e "/auto/mswg/release/ufm/plugins/${Plugin_name}/ufm-plugin-${Plugin_name}_${PLUGIN_VERSION}*" ];then 
+        echo -e "A path and plugin with this version already exist."
```

Comment:
please print the full path.
otherwize you will spend years by guessing what is existing:)